dotnet/src/SemanticKernel/Orchestration/SKFunctionExtensions.cs

```diff
@@ -16,17 +15,6 @@ namespace Microsoft.SemanticKernel.Orchestration;
 /// </summary>
 public static class SKFunctionExtensions
 {
-    /// <summary>
```

Comment:
I know this wasn't used internally or in examples but it is public right now. Is there a need to remove this?

Reply:
Oh nvm i see now.